### PR TITLE
changelog for release 04.25

### PR DESCRIPTION
**Pull request by hsubramanianaks**
This PR is to update B2K CLI to tested version with changelog for release.

### CODE REVIEW

**Review by bfoley13 — APPROVED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,16 @@
 # Change Log
 
+## [2.0.120230418]
+- [Enable auto review by chatgpt](https://github.com/Azure/Bridge-To-Kubernetes/pull/232)
```
Comment (Tatsinnit) [inline CHANGELOG.md:4]: ☕️ thank you for this, and I might have completely missed this last time but all these changes are in B2K repo, so we can refer the release email with a link saying, this **vscode** release is simply the release of B2K new cli. 

Thanks heaps, let’s talk soon as well.

Comment (hsubramanianaks) [reply CHANGELOG.md:4]: agreed, I thought the same thing. I will update them.

Comment (Tatsinnit) [reply CHANGELOG.md]: ❤️☕️ Thank you for the quick loop back, done.

**Review by Tatsinnit — APPROVED** [review]
🎊🚀☕️❤️

**Pull request merged**